src/validators/v1/projects.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
 		if (req.method != common.REQUEST_METHOD_DELETE) {
 			req.checkBody('title').notEmpty().withMessage('title is required')
 		}
-    
+
```

Comment:
@adithyadinesh0412  add validator for submitForReview also

Reply:
Can I add this in the other PR? @priyanka-TL 
this is PR for up for review

Reply:
Can you add in this PR ? Because this only will merge first